app/api/admin.py

```
@@ -212,55 +213,58 @@ def export_json(db: Session = Depends(get_db)) -> StreamingResponse:
 
 
 @router.post("/delete-test-data/")
-async def delete_test_data(db: Session = Depends(get_db)) -> dict[str, int]:
+async def delete_test_data(db: AsyncSession = Depends(get_db)) -> dict[str, int]:
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)